### PR TITLE
fix(allowanceMenu): reword change of budget #805

### LINKED ISSUE #805: Rename the "Edit Allowance" window
**Is your feature request related to a problem? Please describe.**
If I adjust the budget on this site, the whole budget is reset and does not keep the old state. 

**Describe the solution you'd like**
It would be nice to know that the budget is reset in advance (e.g.: "Set a new budget" instead of "Budget").

**Describe alternatives you've considered**
 n.a.

**Additional context**
image

**Are you working on this? (Yes/No)**
No

### PR DESCRIPTION
**Pull request by escapedcat**
#### TODOs

- [ ] ~use translations (Waiting for #906)~

#### Link this PR to an issue
Fixes #805

#### Type of change (Remove other not matching type)

- Bug fix (non-breaking change which fixes an issue)

#### Describe the changes you have made in this PR -

Renamed headline.  
Added more info-text.

#### Screenshots of the changes
image

#### How Has This Been Tested?

Manually tested and added new unit test.

#### Checklist:

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] New and existing tests pass locally with my changes
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

**Comment by escapedcat** [comment]
@MoritzKa what do you think about the added *hint*-text?

**Review by secondl1ght — APPROVED** [review]
I thought this screen was already pretty self-explanatory but LGTM!

File: src/app/components/AllowanceMenu/index.tsx [hunk 1]
```diff
@@ -103,6 +105,9 @@ function AllowanceMenu({ allowance, onEdit, onDelete }: Props) {
               onChange={(e) => setBudget(e.target.value)}
             />
           </div>
+          <p className="my-1 dark:text-white">
+            This will reset the current budget
```
Comment (escapedcat) [inline src/app/components/AllowanceMenu/index.tsx:109]: @dylancom you think we should an optional *hint*-text to inputs?

Comment (dylancom) [reply src/app/components/AllowanceMenu/index.tsx:108]: Nice idea. In italic / a lighter grey for e.g.?

**Review by dylancom — CHANGES_REQUESTED** [review]

**Review by bumi — APPROVED** [review]

File: src/app/components/AllowanceMenu/index.tsx (hunk 1 above)
Comment (escapedcat) [reply src/app/components/AllowanceMenu/index.tsx:108]: image

**Review by dylancom — APPROVED** [review]

**Pull request merged**